Sources/Sentry/SentryHub.m

```diff
@@ -32,12 +32,20 @@ - (instancetype)initWithClient:(SentryClient *_Nullable)client andScope:(SentryS
 - (void)startSession {
     SentrySession *lastSession = nil;
     SentryScope *scope = [self getScope];
+    SentryClient *client = [self getClient];
+    SentryOptions *options = [client options];
+    if (nil == options || nil == options.releaseName) {
+        [SentryLog logWithMessage:[NSString stringWithFormat:@"No option or release to start a session."] andLevel:kSentryLogLevelDebug];
+        return;
```

Comment:
But is it really so bad to have the `releaseName` missing? Isn't it better to have a session without a `releaseName` than to have no session? If the options work fine, the `releaseName` is set to default, if the user doesn't set it. If the user sets the `releaseName` to `nil` manually then sessions would break, if I am not mistaken. So for me it would maybe make sense to just set `releaseName` no matter what.

Reply:
@philipphofmann the session protocol requires a `release` version.